### PR TITLE
Readme improvements

### PR DESCRIPTION
**Pull request by matkoniecz**
- explicitly mention that it is an OSM editor
- use shorter sentences
- be more specific that 'provides the tools' that sounds more complex than reality
- simplify text about economic data use (and clarify that it is achieved)
-----

My work on this pull request and UX testing was sponsored by a NGI Zero Discovery [grant](https://www.openstreetmap.org/user/Mateusz%20Konieczny/diary/368849)

### CODE REVIEW

**Comment by westnordost** [comment]
An improvement, thank you. It could be that this first sentence is very similar to the text found on Google Play and F-Droid

**Comment by matkoniecz** [comment]
> It could be that this first sentence is very similar to the text found on Google Play and F-Droid 

I am not sure what I understand correctly.

Is it "maybe update also https://github.com/westnordost/StreetComplete/blob/master/metadata/en/full_description.txt " or "modify readme further to be closer to https://github.com/westnordost/StreetComplete/blob/master/metadata/en/full_description.txt "?

**Comment by westnordost** [comment]
Rather, **consider maybe** updating the full_description if you feel that the text is too complicated. Though, the audience is also different from the readme.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ![StreetComplete](http://www.westnordost.de/streetcomplete/featureGraphic.png)
 
-An Android app which finds wrong, incomplete or extendable data in the user's vicinity and provides the tools to complete these easily and directly on site without having to use another editor.
+StreetComplete is an easy to use editor of OpenStreetMap data available for Android. It can be used without any OpenStreetMap-specific knowledge. It asks simple questions, with answers directly used to edit and improve OpenStreetMap data.
```
Comment (rugk) [inline README.md]: Maybe link to https://www.openstreetmap.org though? (In case someone discovers it, who does not know what OSM is?)

**Pull request merged**